### ISSUE TITLE
the bot stopped working.

### ISSUE THREAD

**Issue by ruuhkaaika** [tedztar/mcstatusbot#235]
The bot first stopped to update the amount of players on the server and then it stopped responding to any commands and just says "The application did not respond". I tried deleting the bot and adding it back on the server, but it still isn't working..

**Comment by RahulR100** [comment]
Thanks for letting me know! Apologies for the downtime I just got back from a nice long road trip. I have restarted the bot and it should come back online shortly.

**Comment by ruuhkaaika** [comment]
No worries at all, thank you for answering so quickly :)

**Comment by JotaRata** [comment]
Hey can I re-open this real quick?
It seems the issue has returned and I'm getting a "The application did not respond" message on every command

**Comment by RahulR100** [comment]
You sure can! Thanks for letting me know. I have rebooted the server so everything should come back to normal pretty soon. I will also investigate why the systems for downtime notification didn't kick in as it should have.

**Comment by JotaRata** [comment]
you sir.. are the goat 🐐

**Comment by JotaRata** [comment]
This are the only logs I could get on my side.. I don't know if it's helpful or not..

```
[MessageQueue] Draining message from queue LogId:undefined QueueLength: 0
web.e0be055….js:12 [MessageQueue] Draining Message Queue with:  2
web.e0be055….js:12 [MessageQueue] Finished draining message from queue LogId:undefined QueueLength: 0
web.e0be055….js:12 [Flux] Dispatching MESSAGE_SEND_FAILED
```

**Comment by RahulR100** [comment]
Yup those logs seem logical. When the bot (or in this case this one particular shard) goes down, the bot catches it and attempts to restart the problematic shard (subprocess of 1k servers). For some reason, the old shard is not killed properly and causes conflicts when the bot tries to restart it, and results in the shard entering what I can only describe as a zombie state where its active, but does not accept network requests, hence the MESSAGE_SEND_FAILED.

Unsure what causes this at the moment, this will require a bit more thorough investigation.

**Comment by arand4** [comment]
Is this still an issue? on my discord server I've got the same issue, it just says "The application did not respond" on every command.
